src/auth/user-record.ts

```diff
@@ -187,6 +188,7 @@ export class UserRecord {
       validAfterTime = parseDate(response.validSince * 1000);
     }
     utils.addReadonlyGetter(this, 'tokensValidAfterTime', validAfterTime || undefined);
+    utils.addReadonlyGetter(this, 'tenantId', response.tenantId);
```

Comment:
We can directly assign to `this.tenantId` here, but since we've already used the utils method in this constructor, it's ok to leave it as is.

Reply:
Let's make the change in a separate PR.